golem-cli/src/oss/clients/api_definition.rs

```diff
@@ -82,13 +83,13 @@ async fn create_or_update_api_definition<
             Ok(client.import_open_api(&value).await?)
         }
         Action::Create => {
-            let value: HttpApiDefinition = serde_json::from_str(definition_str.as_str())
+            let value: HttpApiDefinitionRequest = serde_json::from_str(definition_str.as_str())
                 .map_err(|e| GolemError(format!("Failed to parse HttpApiDefinition: {e:?}")))?;
 
             Ok(client.create_definition(&value).await?)
         }
         Action::Update => {
-            let value: HttpApiDefinition = serde_json::from_str(definition_str.as_str())
+            let value: HttpApiDefinitionRequest = serde_json::from_str(definition_str.as_str())
```

Comment:
I wouldn't rename something in a PR that's concerned with something else unless if it's mandatory.

Reply:
this is related to OpenAPI spec changes